### ISSUE TITLE
Use assertTrue, assertFalse, assertIsNone in tests (A502)

### ISSUE THREAD

**Issue by peterjc** [biopython/biopython#3278]
This is a spin out from #2700 and #2835, on a related set of stylistic improvements to our use of the ``unittest`` testing framework, which can be spotted using ``flake8-assertive``:

```
$ pip install flake8-assertive
$ cd Tests/
$ flake8 --isolated --select A502
./test_SCOP_Scop.py:70:9: A502 prefer assertIsNone() instead of comparing to None
./test_SCOP_Scop.py:72:9: A502 prefer assertIsNone() instead of comparing to None
./test_SCOP_Scop.py:129:9: A502 prefer assertIsNone() instead of comparing to None
./test_SCOP_Scop.py:133:9: A502 prefer assertIsNone() instead of comparing to None
./test_Phylo.py:394:9: A502 prefer assertFalse() instead of comparing to False
./test_PDB_MMCIFParser.py:324:9: A502 prefer assertIsNone() instead of comparing to None
./test_PDB_MMCIFParser.py:336:9: A502 prefer assertIsNone() instead of comparing to None
./test_PDB_Polypetide.py:79:9: A502 prefer assertIsNone() instead of comparing to None
./test_PDB_Polypetide.py:87:9: A502 prefer assertIsNone() instead of comparing to None
./test_PDB_Polypetide.py:95:9: A502 prefer assertIsNone() instead of comparing to None
./common_BioSQL.py:704:13: A502 prefer assertFalse() instead of comparing to False
./test_phenotype.py:135:9: A502 prefer assertTrue() instead of comparing to True
./test_phenotype.py:136:9: A502 prefer assertFalse() instead of comparing to False
./test_phenotype.py:259:9: A502 prefer assertIsNone() instead of comparing to None
./test_phenotype.py:260:9: A502 prefer assertIsNone() instead of comparing to None
./test_phenotype.py:261:9: A502 prefer assertIsNone() instead of comparing to None
./test_phenotype.py:262:9: A502 prefer assertIsNone() instead of comparing to None
./test_phenotype.py:263:9: A502 prefer assertIsNone() instead of comparing to None
./test_phenotype.py:264:9: A502 prefer assertIsNone() instead of comparing to None
./test_phenotype.py:265:9: A502 prefer assertIsNone() instead of comparing to None
./test_PhyloXML.py:169:9: A502 prefer assertTrue() instead of comparing to True
./test_PhyloXML.py:174:9: A502 prefer assertFalse() instead of comparing to False
./test_PhyloXML.py:721:9: A502 prefer assertTrue() instead of comparing to True
./test_PhyloXML.py:722:9: A502 prefer assertFalse() instead of comparing to False
./test_PDB_PDBIO.py:237:13: A502 prefer assertIsNone() instead of comparing to None
./test_Restriction.py:151:9: A502 prefer assertIsNone() instead of comparing to None
./test_Restriction.py:568:9: A502 prefer assertIsNone() instead of comparing to None
./test_SeqIO_index.py:119:13: A502 prefer assertTrue() instead of comparing to True
./test_SeqIO_index.py:213:13: A502 prefer assertTrue() instead of comparing to True
./test_SeqIO_index.py:669:13: A502 prefer assertTrue() instead of comparing to True
./test_Entrez_parser.py:2294:9: A502 prefer assertIsNone() instead of comparing to None
./test_Entrez_parser.py:2303:9: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:13946:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:13978:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14010:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14107:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14143:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14255:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14302:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14400:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14500:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14519:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14540:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14564:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14583:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14604:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14628:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14666:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14690:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14709:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14730:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14754:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14773:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14794:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14818:13: A502 prefer assertIsNone() instead of comparing to None
./test_NCBITextParser.py:14838:13: A502 prefer assertIsNone() instead of comparing to None
./test_PDB_PDBParser.py:113:9: A502 prefer assertIsNone() instead of comparing to None
./test_PQR.py:38:9: A502 prefer assertIsNone() instead of comparing to None
./test_PQR.py:39:9: A502 prefer assertIsNone() instead of comparing to None
./test_PQR.py:69:9: A502 prefer assertIsNone() instead of comparing to None
./test_PQR.py:77:9: A502 prefer assertIsNone() instead of comparing to None
./test_PQR.py:102:9: A502 prefer assertIsNone() instead of comparing to None
./test_PQR.py:110:9: A502 prefer assertIsNone() instead of comparing to None
./test_PQR.py:118:9: A502 prefer assertIsNone() instead of comparing to None
./test_Application.py:43:9: A502 prefer assertIsNone() instead of comparing to None
./test_Application.py:50:9: A502 prefer assertIsNone() instead of comparing to None
./test_Application.py:55:9: A502 prefer assertIsNone() instead of comparing to None
./test_Application.py:56:9: A502 prefer assertIsNone() instead of comparing to None
./test_Application.py:65:9: A502 prefer assertIsNone() instead of comparing to None
./test_Application.py:78:9: A502 prefer assertIsNone() instead of comparing to None
./test_Application.py:92:9: A502 prefer assertIsNone() instead of comparing to None
./test_Application.py:93:9: A502 prefer assertIsNone() instead of comparing to None
./test_Application.py:109:9: A502 prefer assertIsNone() instead of comparing to None
./test_Application.py:110:9: A502 prefer assertIsNone() instead of comparing to None
./test_Affy.py:72:13: A502 prefer assertTrue() instead of comparing to True
./test_Affy.py:75:13: A502 prefer assertFalse() instead of comparing to False
./test_Affy.py:78:13: A502 prefer assertTrue() instead of comparing to True
./test_Affy.py:81:13: A502 prefer assertFalse() instead of comparing to False
./test_Affy.py:82:13: A502 prefer assertFalse() instead of comparing to False
```

I'm marking this as a good first issue, and suggest people wanting to work on this comment here on this issue saying which file(s) they will fix first to avoid duplication of effort.

**Comment by vinisalazar** [comment]
Hi, I'd like to work on this! Is that okay? I've read the contribution guidelines and will commence local testing soon.

**Comment by peterjc** [comment]
@vinisalazar yes please - just comment here on which file(s) you'll be starting with.

**Comment by vinisalazar** [comment]
I should follow the order listed here.

**Comment by vinisalazar** [comment]
Addressed by #3279

**Comment by peterjc** [comment]
And that's merged, thank you!
